lib/conditional-get.test.js

```diff
@@ -12,22 +12,20 @@ describe('Conditional-Get', () => {
 
     const server = app.listen();
 
-    test('should set an etag', done => {
-      request(server)
+    test('should set an etag', () => {
+      return request(server)
         .get('/')
-        .expect(200, (err, res) => {
-          if (err) return done(err);
```

Comment:
Yes, because if supertest throws an error the promise will reject, and since it's returned to Jest it will be automatically handled and the test will error.